run_calculate.py

```diff
@@ -18,7 +18,7 @@
 if __name__ == "__main__":
 
     # test with dummy data
-    business_trip_data = glob.glob(f"{script_path}/data/test_data_users/business_trips*.csv")
+    business_trip_data = glob.glob(f"{script_path}/data/test_data_users/business_trips_plane_goethe.csv")
```

Comment:
Should we maybe remove this from the PR @veitu ?

Reply:
I changed it back